config.lisp

```diff
@@ -57,8 +70,9 @@
 			    (intern (symbol-name (first (first specs)))
 				    (find-package '#:cl-migrations)))
 		(second (first specs)))
-	  (setf *migration-dir* (first (second (second specs))))
-	  (unless (equal (subseq *migration-dir* (1- (length *migration-dir*))) "/")
+	  (setf *migration-dir* (resolve-migration-dir specs))
+	  (unless (or (null *migration-dir*)
+                      (equal (subseq *migration-dir* (1- (length *migration-dir*))) "/"))
```

Comment:
Sorry for the noise: I was sure we had alexandria in the dependencies (of cl-sql), but we don't.